app/Config/CURLRequest.php

```diff
@@ -6,6 +6,22 @@
 
 class CURLRequest extends BaseConfig
 {
+    /**
+     * --------------------------------------------------------------------------
+     * CURLRequest Share Connection
+     * --------------------------------------------------------------------------
+     *
+     * Whether share connection between requests.
```

Comment:
The sentence suggests it is boolean rather than as array. Please revise. I think this property should read as "shareConnectionOptions".